presto-hive/src/main/java/com/facebook/presto/hive/HiveSplitManager.java

```diff
@@ -250,6 +257,12 @@ public CounterStat getHighMemorySplitSource()
                 // verify all partition is online
                 verifyOnline(tableName, Optional.of(partName), getProtectMode(partition), table.getParameters());
 
+                // verify none of the partition has OBJECT_NOT_READABLE flag
```

Comment:
Let's change this to
```java
// verify partition is not marked as non-readable
```
This describes what is being done, rather than the implementation details, and is singular since only one partition is being checked here.